swift/BUILDING.md

```diff
@@ -20,36 +20,31 @@ resulting build.
 Ice for Swift builds and runs on macOS and is supported on the platforms listed
 on the [supported platforms] page.
 
-### Slice to Swift Compiler
-
-You need the Slice to Swift compiler to build Ice for Swift and also to use Ice
-for Swift. The Slice to Swift compiler (`slice2swift`) is a command-line tool
-written in C++. You can build the Slice to Swift compiler from source, or
-alternatively you can install an Ice [binary distribution] that includes this
-compiler.
-
 ### Swift Version
 
 Ice for Swift requires Swift 5 or later.
 
 ## Building Ice for Swift
 
-Then open `ice.xcodeproj` with Xcode and build the `Ice macOS` or `Ice iOS`
-targets.
+Building Ice for Swift requires having built Ice for C++ first from the `cpp` directory. This builds the required
+`XCFramework`s that are used by the Swift package.
```

Comment:
Meaning the Swift package requires multiple XCFrameworks?

It would be clearer to write something:
This builds the 3 C++ frameworks that are used by the Swift package.

Reply:
It's actually one nested XCFramework.

Reply:
I see 3 separate XCFramework:

```
lib/Ice.xcframework
lib/IceDiscovery.xcframework
lib/IceLocatorDiscovery.xcframework
```
Where is the nested XCFramework?

Reply:
Right, I forgot about Ice*Discovery. There's a platform dependent xcframework for each of the three platforms we support inside each of those frameworks.